components/Cta/CtaGroup.tsx

```diff
@@ -17,6 +17,7 @@ export const CtaGroup = ({
     <FlexBox
       {...props}
       as="ul"
+      mb="1em" // Gatsby build adds a 1em margin-bottom to each cta inside a CTA group
```

Comment:
Flex gap didn't get universal support when Giving was build, so there was a 1em margin bottom that was added to all the CTAs inside the CTA group at the time. This adds the backward compatibility